hearth/lib/hearth/http/client.rb

```diff
@@ -12,31 +12,31 @@ class Client
       #
       # @param [Hash] options The options for this HTTP Client
       #
-      # @option options [Boolean] :http_wire_trace (false) When `true`,
-      #   HTTP debug output will be sent to the `:logger`.
+      # @option options [Boolean] :debug_output (false) When `true`,
+      #   HTTP debug output will be sent to the configured logger.
       #
       # @option options [Logger] :logger A logger where debug output is sent.
       #
-      # @option options [String] :http_proxy A proxy to send
+      # @option options [String] :proxy A proxy to send
       #   requests through. Formatted like 'http://proxy.com:123'.
       #
-      # @option options [Boolean] :ssl_verify_peer (true) When `true`,
+      # @option options [Boolean] :verify_peer (true) When `true`,
       #   SSL peer certificates are verified when establishing a
       #   connection.
       #
-      # @option options [String] :ssl_ca_bundle Full path to the SSL
+      # @option options [String] :ca_file Full path to the SSL
```

Comment:
the`ssl_` prefix feels a bit different than the `http` prefix - seems like these may still be useful in providing clarity on what its for?

Reply:
These were changed to exactly what Net::HTTP uses. I think the prefixes made sense when they were public configuration. Now Client is the only public config.